packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -404,6 +404,28 @@ class AdaptiveScaffold extends StatefulWidget {
     );
   }
 
+  /// Animation from top off the screen into the screen.
+  static AnimatedWidget topOutIn(Widget child, Animation<double> animation) {
```

Comment:
We might need to rethink the names here (yours, and the existing ones) because the naming should be consistent, and the existing naming doesn't really distinguish between which side of the screen is the start and the destination, and how it gets there.

Names that are more descriptive as to the type of animation, and where it goes would make more sense. For instance "slideOffToTop" or "slideOnFromTop" might be better for these two. The symbols `topToBottom` and `bottomToTop` might be better as  `slideOnFromBottom` and `slideOffToBottom`, etc.

Reply:
I think it would be better too. How about this?

- horizontal
  -  `slideOnFromTop` `slideOffToTop` `slideOnFromBottom` `slideOffToBottom`
- vertical
  - `slideOnFromLeft` `slideOffToLeft` `slideOnFromRight` `slideOffToRight`

And mark the existing animations as `@Deprecated`.

Reply:
Yes, that works, and you'll need to bump the minor version number instead of the patch number, since people who have deprecations as errors will have their compile break.

Reply:
Done. Use version 0.2.0.